timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -272,12 +272,18 @@ export class TimeGraphChart extends TimeGraphChartLayer {
                 this.mouseEndX = e.offsetX;
                 const start = this.mouseZoomingStart;
                 const end = this.unitController.viewRange.start + (this.mouseEndX / this.stateController.zoomFactor);
-                if (start !== end) {
```

Comment:
I think we need to keep this check, if the user just right-clicks without moving, we want to ignore it and not zoom to 1ns.

As a user (but because I know the implementation) sometimes I abort my zooming by going back to the start pixel...

Reply:
Updated. Thanks.